client/components/Nav.jsx

```diff
@@ -820,7 +819,8 @@ Nav.defaultProps = {
   warnIfUnsavedChanges: undefined,
   params: {
     username: undefined
-  }
+  },
+  language: 'en-US'
```

Comment:
I don't think we should hard-code `en-US` here as it introduces duplication throughout the codebase. Let's make language required as the Redux store will be initialised with a default language.

Reply:
Ok, I did it to remove a warning that is received when you run
 ``` npm run test ```
 with result : 
``` Warning: Failed prop type: The prop `language` is marked as required in `Nav`, but its value is `undefined`.``` 
but the tests pass anyway

Reply:
If there's a warning in the tests then that needs to be fixed.

It's happening because `language` is required and it's being being supplied. Try adding a default language (`en-US`) to [the `props` object](https://github.com/processing/p5.js-web-editor/blob/develop/client/components/__test__/Nav.test.jsx#L48-L50).

Reply:
Correct, your suggestion worked. Thanks.